agent/handlers/v4/container_metadata_handler.go

```diff
@@ -65,6 +69,7 @@ func ContainerMetadataHandler(state dockerstate.TaskEngineState) func(http.Respo
 func GetContainerResponse(containerID string, state dockerstate.TaskEngineState) (*ContainerResponse, error) {
 	containerResponse, err := NewContainerResponse(containerID, state)
 	if err != nil {
+		seelog.Errorf("unable to get container metadata for container '%s'", containerID)
 		return nil, errors.Errorf("unable to generate metadata for container '%s'", containerID)
```

Comment:
shouldnt this be returning TransientError?

Reply:
I think it should still be a Bad Request error., if it is unable to find container metadata by containerID or TaskID?